cli/azd/internal/repository/app_init.go

```diff
@@ -432,3 +455,14 @@ func prjConfigFromDetect(
 
 	return config, nil
 }
+
+func countPrefix(s string, prefix string) int {
+	count := 0
+	for strings.HasPrefix(s, prefix) {
+		count++
+		// len(s) >= len(prefix) guaranteed by HasPrefix
+		s = s[len(prefix):]
+	}
+
+	return count
+}
```

Comment:
Adding coverage for this method should be trivial.. even a task for copilot :)

Reply:
I do have this and wrote this as well with the initial implementation :). But I also feel like in some sense it's a waste of text space once it can be trivially verified.